scripts/check-ts.js

```diff
@@ -59,9 +59,38 @@ if (filterPackage) {
   }
 }
 
+let totalTsFiles = 0
+let totalJsFiles = 0
+
 packagesWithTs.forEach(project => {
+  const tsFiles = glob.sync(
+    `./packages/${project.split(/.*packages[/\\]/)[1]}/src/**/*.ts`,
+    {
+      root: project,
+      ignore: `**/node_modules/**`,
+    }
```

Comment:
Both options seem unnecessary here? 

- `root` only affects patterns starting with `/`
- `ignore` also never used as we only look into `src` folder which doesn't have `node_modules`